cumulusci/cli/cci.py

```diff
@@ -600,8 +600,17 @@ def project_init(runtime):
     # Render templates
     for name in (".gitignore", "README.md", "cumulusci.yml"):
         template = env.get_template(name)
-        with open(name, "w") as f:
-            f.write(template.render(**context))
+        if not os.path.isfile(name):
```

Comment:
I'd recommend using Pathlib's [is_file()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file) method instead of the `os` module for stuff like this.